src/network/MissionControlProtocol.cpp

```diff
@@ -12,6 +12,7 @@
 #include <functional>
 #include <loguru.hpp>
 #include <mutex>
+#include <optional>
```

Comment:
Again here - is `optional` inclusion necessary? It's included in world_interface/data.h.